### PR TITLE
docs(readme): update certification details

### LINKED ISSUE #39170: Update readme on the repository
@QuincyLarson 

We would want to update our [readme](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/README.md) to the current certifications.

Assigning to you because you are the best suited for this update.

### PR DESCRIPTION
**Pull request by riyasavant**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #39170

The README.md file hadn't been updated with the latest Python Certification courses. Changed the file to add links to the respective course sections and other relevant changes.

### CODE REVIEW

File: README.md
Comment (nhcarrigan) [inline README.md:136]: If we are updating the certifications, the Full Stack certification could use some clarification as well.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: README.md
Comment (riyasavant) [reply README.md]: I have committed those changes, I'm not sure how does that get updated on the PR.

Comment (nhcarrigan) [reply README.md]: When you commit the changes to your branch, it automatically updates the PR as well. 🙂

Comment (riyasavant) [reply README.md]: Could you please check out the new changes made as requested?

File: README.md
Comment (nhcarrigan) [inline README.md:136]:
```suggestion
Once you have earned the Responsive Web Design, Algorithms and Data Structures, Front End Libraries, Data Visualization, APIs and Microservices, and Legacy Information Security and Quality Assurance certifications, you'll be able to claim your freeCodeCamp.org Full Stack Development Certification. This distinction signifies that you've completed around 1,800 hours of coding with a wide range of web development tools.
```

I think we should specifically mention the requirements for the Full Stack certification, and remove the mention of the python certifications.

File: README.md
Comment (nhcarrigan) [inline README.md:134]:
```suggestion
#### Legacy Full Stack Development Certification
```

Modify this title.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
I think it would work best to specifically mention just the Full Stack certification (the python certifications do not have one overarching certificate). 

I'm also on the fence about whether this should be updated or removed entirely - would be open to feedback from others.

**Comment by riyasavant** [comment]
> I'm also on the fence about whether this should be updated or removed entirely - would be open to feedback from others.

I prefer the update, as not many are aware of the Legacy Certification provided.

**Review by naomi-lgbt — DISMISSED** [review]
The changes look good to me! Congratulations on opening your first pull request! 🥳 

A couple of notes:
- Our workflow requires a second approving review before your changes can be merged.
- When working on changes, it is best to work off of a new branch so that your master branch remains clean.

**Comment by riyasavant** [comment]
Awesome! Will keep that in mind 🙂

**Comment by RandellDawson** [comment]
Since the `README.md` is highly visible, @QuincyLarson should review this PR also.

File: README.md
Comment (QuincyLarson) [inline README.md:12]: The M in machine learning shouldn't be capitalized here. And there's no need to say "with Python"

**Review by QuincyLarson — COMMENTED** [review]
@riyasavant Thanks for opening this pull request. I just had one minor edit. Everything else looks great. Once you've made this edit, we can merge it.

File: README.md
Comment (riyasavant) [reply README.md]: Updated the requested changes.

**Review by ahmaxed — APPROVED** [review]

**Review by naomi-lgbt — APPROVED** [review]
LGTM!

**Pull request merged**
